numba/npyufunc/wrappers.py

```diff
@@ -328,6 +334,17 @@ def _build_wrapper(self, library, name):
 
         fnty = Type.function(Type.void(), [byte_ptr_ptr_t, intp_ptr_t,
                                            intp_ptr_t, byte_ptr_t])
+        return fnty
+
+    def _build_wrapper(self, library, name):
+        """
+        The LLVM IRBuilder code to create the gufunc wrapper.
+        The *library* arg is the CodeLibrary for which the wrapper should
+        be added to.  The *name* arg is the name of the wrapper function being
```

Comment:
Perhaps `The *library* arg is the CodeLibrary to which the wrapper should be added.`